internal/datadog/go.mod

```
@@ -0,0 +1,72 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/internal/datadog
+
+go 1.19
+
+require (
+	github.com/DataDog/datadog-agent/pkg/trace v0.46.0
+	github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/metrics v0.5.2
+	github.com/stretchr/testify v1.8.4
+	go.opentelemetry.io/collector/pdata v1.0.0-rcv0013
```

Needs a bump
```suggestion
	go.opentelemetry.io/collector/pdata v1.0.0-rcv0014
```

and `make gotidy` afterwards

---

This isn't resolved?